core/templates/dev/head/pages/topic_editor/subtopics_editor/SubtopicsListTabDirective.js

```diff
@@ -27,27 +27,91 @@ oppia.directive('subtopicsListTab', [
       controller: [
         '$scope', '$uibModal', 'TopicEditorStateService', 'TopicUpdateService',
         'UndoRedoService', 'EVENT_TOPIC_REINITIALIZED',
-        'EVENT_TOPIC_INITIALIZED',
+        'EVENT_TOPIC_INITIALIZED', 'EVENT_SUBTOPIC_PAGE_LOADED',
         function(
             $scope, $uibModal, TopicEditorStateService, TopicUpdateService,
             UndoRedoService, EVENT_TOPIC_REINITIALIZED,
-            EVENT_TOPIC_INITIALIZED) {
+            EVENT_TOPIC_INITIALIZED, EVENT_SUBTOPIC_PAGE_LOADED) {
+          $scope.subtopicDisplayed = false;
           var _initEditor = function() {
             $scope.topic = TopicEditorStateService.getTopic();
             $scope.subtopics = $scope.topic.getSubtopics();
+            $scope.previewSubtopicPage = true;
```

Comment:
This seems a bit odd to me. I would do it more like `$scope.subtopicEditorIsShown = false;`, since otherwise this suggests some kind of "separate" preview view.

Reply:
Done.